plugins/vpp/model/api_version.go

```diff
@@ -0,0 +1,18 @@
+//  Copyright (c) 2018 Cisco and/or its affiliates.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at:
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package model
+
+// ProtoApiVersion is current version of the northbound api
+const ProtoApiVersion = "/v1/"
```

Comment:
If we want to keep similar parts of prefix in single place, perhaps we could define `vpp/config/` and `vpp/status/` here.